docs/src/main/asciidoc/web.adoc

```diff
@@ -0,0 +1,273 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+= Quarkus for the Web
+include::_attributes.adoc[]
+:categories: web
+:summary: Learn more about creating all kinds of Web applications with Quarkus.
+:numbered:
+:sectnums:
+:sectnumlevels: 3
+:topics: http,web,renarde,full-stack,qute,quinoa,web-bundler,mvc,ssr,nodejs,npm,javascript,css,jsf
```

Comment:
this should be jsf and faces. With Faces 4.0 Jakarta is trying to get away from the old JSF naming to Faces.